presto-orc/src/main/java/com/facebook/presto/orc/StripeReader.java

```diff
@@ -150,6 +150,7 @@ public Stripe readStripe(
             SharedBuffer sharedDecompressionBuffer)
             throws IOException
     {
+        runtimeStats.addMetricValue("stripeCount", 1);
```

Comment:
Should we keep all metrics key as a constants in one file? It would easier to refer all constants then?

Reply:
We do have such a file for metrics in added in presto-main, e.g. the operator metrics and task metrics.
But it is hard to enforce that for metrics exposed from connectors as these could be exposed from a different repo. E.g., some metrics are exposed from Alluxio repo. And we don't want the connector developers to update a file in presto-main for a metric change in the connector.

Besides, these metrics will be included in the operator summary which is specific to one operator. So name conflict may not be a big concern.

Reply:
Added Orc prefix to the counter names to make it more clear.